test/old/testdir/test_ins_complete.vim

```diff
@@ -2334,7 +2334,13 @@ func Test_thesaurusfunc_callback()
     new
     call setline(1, 'sun')
     LET g:TsrFunc1Args = []
-    call feedkeys("A\<C-X>\<C-T>\<Esc>", "x")
+    " Nvim: message is an actual error rather than message with error highlight
+    try
+      call feedkeys("A\<C-X>\<C-T>", "x")
+    catch
+      call assert_exception("'thesaurus' option is empty")
+      call feedkeys("\<Esc>", "x")
+    endtry
```

Comment:
The `x` flag will automatically add `<Esc>` if still in Insert mode when the keys end, so moving the `<Esc>` to a separate `feedkeys()` call is wrong. Plus, maybe it's better to use just this?
```suggestion
    silent! call feedkeys("A\<C-X>\<C-T>\<Esc>", "x")
```